app/models/health/tasks/import_epic.rb

```diff
@@ -133,10 +133,10 @@ def update_housing_statuses
 
     # keep pilot patients in sync with epic export
     def sync_epic_pilot_patients
-      referral_patients = Health::Patient.bh_cp.pluck(:medicaid_id)
+      patients_by_datasource = Health::Patient.pluck(:medicaid_id, :data_source_id).to_h
```

Comment:
Isn't the `to_h` going to mean you only get one of them?  (duplicate medicaid IDs).  Or is the idea, we should never have multiple patients with the same medicaid id?

Reply:
Yes, that is the idea.